enterprise/models.py

```diff
@@ -261,6 +262,13 @@ class Meta:
         default='',
         help_text=_("Hostname of the enterprise learner portal, e.g. bestrun.edx.org.")
     )
+    catalog_uuids = models.TextField(
+        blank=True,
+        null=True,
+        default=[],
```

Comment:
default set to `[]` for Textfield. we probably don't need a default since it is nullable field.